pkg/cmd/cli/cmd/rsh.go

```diff
@@ -143,5 +144,10 @@ func (o *RshOptions) Validate() error {
 
 // Run starts a remote shell session on the server
 func (o *RshOptions) Run() error {
+	// Insert the TERM into the command to be run
+	// TODO what if the container doesn't have `env`???
+	term := fmt.Sprintf("TERM=%s", util.Env("TERM", "vt100"))
```

Comment:
If there is to be any default, I'd go with the same value that Docker uses for `docker run`: xterm.
[docker/docker/container/container_unix.go#L61-L63](https://github.com/docker/docker/blob/cbf785c45a8cca061028ec341320bcbd655cf1fd/container/container_unix.go#L61-L63)